src/Microsoft.Cci.Extensions/Microsoft.Cci.Extensions.csproj

```diff
@@ -14,7 +14,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{2179F9B5-1DBA-4563-9402-A94DE75EA9FA}</ProjectGuid>
     <DefineConstants>$(DefineConstants);COREFX</DefineConstants>
-    <NuGetTargetMoniker>.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
```

Comment:
I would prefer we stick with netstandard1.3 so we don't accidently take dependencies on newer APIs as netstandard 1.3 is compatible with net46 as well where as 1.5 is not.